components/Tabs/src/TabBarView/MDCTabBarView.m

```diff
@@ -431,23 +432,53 @@ - (CGFloat)justifiedWidth {
 - (void)scrollUntilSelectedItemIsVisibleWithoutAnimation {
   NSUInteger index = [self.items indexOfObject:self.selectedItem];
   if (index == NSNotFound || index >= self.containerView.arrangedSubviews.count) {
+    index = 0;
+  }
+  if (self.containerView.arrangedSubviews.count == 0U) {
```

Comment:
Nit: Is there a reason why you use 0U here and no format for CGFloats below (in: (CGRectGetWidth(self.containerView.bounds) < 0)...) ?

Reply:
`count` is an unsigned value. So unsigned comparison is required.  Below the return type of `CGRectGet...` functions is `CGFloat`.